app/serializers/course_serializer.rb

```diff
@@ -0,0 +1,26 @@
+class CourseSerializer
+  attr_reader :course, :days
+  private :course, :days
```

Comment:
@KevinMulhern I have never seen this before, can you help me understand what this does?

Reply:
Sure, the `attr_reader` exposes course and days getter methods both internally and externally for this this object. We only want them to be accessible internally though, we don't want them to be part of the objects external API so we can make them private to achieve that.

Reply:
Ah, thanks for the explanation. 

I have always used something like this:

```ruby
class CourseSerializer
  # ...

  private

  attr_reader :course, :days
end